build.sh

```diff
@@ -4,6 +4,19 @@
 
 # Get directory this script is in
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+OS=$(uname -s)
+
+if [ "$OS" = "Darwin" ]; then
+    DIR_OS="MacOS"
+else
+    DIR_OS="Linux"
+fi
```

Comment:
Make this default value for DIR_OS, and combine all 3 other case into single if/elif/elif?

Reply:
No, they are different. Linux or MacOS could be overwritten and the DIR_OS become ios/android.